### PR TITLE
[fix](tvf) fix FE cannot start when replay `alter view from tvf`

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

File: regression-test/suites/external_table_p0/tvf/test_hdfs_tvf.groovy [hunk 1]
```diff
@@ -312,6 +312,28 @@ suite("test_hdfs_tvf","external,hive,tvf,external_docker") {
...
+                """
+
+            sql """
+                alter view test_hdfs_tvf_create_view as
+                select c1 from HDFS(
+                        "uri" = "${uri}",
+                        "hadoop.username" = "${hdfsUserName}",
+                        "column_separator" = ",",
+                        "format" = "${format}") order by c1;
+                """
```
Comment (morningman) [inline regression-test/suites/external_table_p0/tvf/test_hdfs_tvf.groovy:336]: add a basic select case, before and after "alter"

Comment (BePPPower) [reply regression-test/suites/external_table_p0/tvf/test_hdfs_tvf.groovy:338]: done

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Review by yiguolei — DISMISSED** [review]

**Comment by morningman** [comment]
Need to add upgrade test cases

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by wuwenchi — APPROVED** [review]

**Pull request merged**